_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -12,6 +12,14 @@ Packages for Foreman and its various plug-ins are now being built for Ubuntu 20.
 
 Packages for Foreman and its various plug-ins that are built for CentOS 8 are now being tested on CentOS 8 Stream as well.
 
+#### Host registration improvements
+
+The host registration form has been redesigned and has new fields. The `Install Packages` field can be used to install additional packages while the `Repository & GPG keys` fields can be used to configure additional software repositories for yum/dnf or apt.
```

Comment:
I didn't check, but are `Repository` and `GPG Keys` separate fields? If so, I'd be explicit:
```suggestion
The host registration form has been redesigned and has new fields. The `Install Packages` field can be used to install additional packages while the `Repository` and `GPG keys` fields can be used to configure additional software repositories for yum/dnf or apt.
```